Formula/rmw.rb

```diff
@@ -31,7 +31,10 @@ class Rmw < Formula
 
   def install
     mkdir "build" do
-      system "meson", *std_meson_args, ".."
```

Comment:
I had to replace this because it included a flag to disable fallback options. The only fallback option I'm using is for the [canfigger library](https://github.com/andy5995/canfigger/), which was part of rmw, but I split it from the rmw code to create a separate library. What happens now in the rmw meson build is that canfigger is built but linked statically, and the library is not installed when the superproject is installed.

Reply:
Let's not do this -- the flag to disable fallback options is there because we don't want Meson to use fallback options.

We can build the canfigger library separately -- as a `resource` if necessary.

Reply:
I'm sorry, I don't understand. Could you provide a link to the documentation where the problem is explained and has instructions how to fix?

Reply:
This doesn't come up often enough to have made it worth writing documentation for, I'm afraid.

But you can fix this by either:
1. adding a new formula for the canfigger library (especially if other software might use it); or,
2. add a `resource` block to the formula for the canfigger library. The resource block should allow you to build the library separately. We should probably just build a static version of the library if we do this.

You can `grep` the repo for `resource ".*" do` for many examples of this. (Just skip the ones that have a Python dependency; those won't be helpful for you here.)

Reply:
Thanks @carlocab . I've submitted a PR to add canfigger as a new formula in https://github.com/Homebrew/homebrew-core/pull/94085

Reply:
@carlocab I've fixed this up to use canfigger now, without using the fallback. Thanks again for your help.